### PR TITLE
Welcome landing page

### PR DESCRIPTION
**Pull request by Mjaethers**
A landing page for new visitors to the oakestra docs

### CODE REVIEW

**Comment by nitindermohan** [comment]
Rebased onto main brach so that page links are (somewhat) accurate

**Comment by Mjaethers** [comment]
Were any changes made to the landing page? The git history is a little confusing

**Comment by nitindermohan** [comment]
Yes. Please take latest commit

File: content/docs/getting-started/welcome-to-oakestra.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Welcome to Oakestra Documentation"
```
Comment (Mjaethers) [inline content/docs/getting-started/welcome-to-oakestra.md]: I think
```suggestion
title: "Welcome to Oakestra's Documentation"
```
or
```suggestion
title: "Welcome to the Oakestra Documentation"
```
makes more sense.

Comment (nitindermohan) [reply content/docs/getting-started/welcome-to-oakestra.md]: It cuts into the TOC bar on the side.

File: content/docs/getting-started/welcome-to-oakestra.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+If you're looking for some concrete examples to get a an understanding of what's possible with Oakestra then
+the app catalog is the place for you.
+
+{{< link-card
+  title="Want some examples?"
+  description="Check out our App catalog"
+  href="../../manuals/app-catalog/example-applications"
+>}}
+
```
Comment (nitindermohan) [inline content/docs/getting-started/welcome-to-oakestra.md:68]: not clear how to add link to pub page here.

**Comment by nitindermohan** [comment]
@Mjaethers can you please take a look at the page. There is a broken link to publications which I have commented

**Comment by Malyuk-A** [comment]
I am confused and concerned - why is this PR so large and includes unrelated changes/files from PRs that have been merged a long time ago (e.g. the CLI manuals automation components) 
(I currently lack time to dive into this myself)

**Comment by giobart** [comment]
> @Mjaethers can you please take a look at the page. There is a broken link to publications which I have commented

If you refer to the /publications link inside callout i fixed that

File: content/docs/getting-started/welcome-to-oakestra.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+- [Addons](/docs/concepts/oakestra-extensions/addons/)
+- [Hooks](/docs/concepts/oakestra-extensions/hooks/)
+- [Custom Resources](/docs/concepts/oakestra-extensions/custom-resources/)
+{{< /card >}}
+
+{{< card title="Developer Oriented" icon="outline/box-multiple" color="red" >}}
+Leverage several developer-forward Oakestra features to make your applications edge-cloud efficient. 
+
+- [Multi-Virtualization Support](/docs/manuals/execution-runtimes/supported-virtualization-runtimes/)
+- Semantic Networking
```
Comment (giobart) [inline content/docs/getting-started/welcome-to-oakestra.md:58]: Leaving out a reminder to add this link once PR #14 is merged

**Review by giobart — COMMENTED** [review]
image

Can we link to get started high level arch for the next reading?

**Comment by giobart** [comment]
> I am confused and concerned - why is this PR so large and includes unrelated changes/files from PRs that have been merged a long time ago (e.g. the CLI manuals automation components) (I currently lack time to dive into this myself)

Good point... there might be some rebase issues here. I hope this does not overwrite content.

**Comment by Mjaethers** [comment]
> image
> 
> Can we link to get started high level arch for the next reading?

I added an [issue](https://github.com/nitindermohan/oak-new-docs/issues/44) regarding this, are there any more documents in the pipeline?

**Comment by Mjaethers** [comment]
> > I am confused and concerned - why is this PR so large and includes unrelated changes/files from PRs that have been merged a long time ago (e.g. the CLI manuals automation components) (I currently lack time to dive into this myself)
> 
> Good point... there might be some rebase issues here. I hope this does not overwrite content.

I think this is due to using `git pull` instead of `git pull --rebase` when the remote and local branch diverged

**Comment by nitindermohan** [comment]
@Malyuk-A I had to do a rebase with main to make the links available for the landing page. On top of that, there are UI updates in all PRs that were already merged (fixed UI styles, grammar issues, interconnections etc). I do not think there are overwriting artifacts but we can do a side-by-side visual check to make sure

**Comment by nitindermohan** [comment]
@Mjaethers regarding https://github.com/nitindermohan/oak-new-docs/pull/31#issuecomment-2551429345, lets wait for all PRs to be merged as we will keep having these artifacts otherwise

**Comment by Malyuk-A** [comment]
Just to keep this in a close loop:

Here is the squashed/rebased version of this PR
https://github.com/nitindermohan/oak-new-docs/pull/48

**Comment by nitindermohan** [comment]
Overridden by https://github.com/nitindermohan/oak-new-docs/pull/48

**Pull request closed without merging**